Source/WebInspectorUI/UserInterface/Views/OverrideDeviceSettingsPopover.js

```diff
@@ -0,0 +1,324 @@
+
+        return contentElement;
+    }
+
+    // Private
+
+    _createSettingsSection()
+    {
+        let coreSettingsGroup = {
+            name: WI.UIString("Core Features"),
+            settings: [
+                {name: WI.UIString("Disable Images"), setting: InspectorBackend.Enum.Page.Setting.ImagesEnabled, value: false},
+                {name: WI.UIString("Disable CSS"), setting: InspectorBackend.Enum.Page.Setting.AuthorAndUserStylesEnabled, value: false},
+                {name: WI.UIString("Disable JavaScript"), setting: InspectorBackend.Enum.Page.Setting.ScriptEnabled, value: false},
+            ],
+        };
+
+        let compatibilitySettingsGroup = {
+            name: WI.UIString("Compatibility"),
+            settings: [
+                {name: WI.UIString("Disable site-specific hacks"), setting: InspectorBackend.Enum.Page.Setting.NeedsSiteSpecificQuirks, value: false},
+            ],
+        };
+
+        let securitySettingsGroup = {
+            name: WI.UIString("Security"),
+            settings: [
+                {name: WI.UIString("Disable cross-origin restrictions"), setting: InspectorBackend.Enum.Page.Setting.WebSecurityEnabled, value: false},
+            ],
+        };
+
+        let privacySettingsGroup = {
+            name: WI.UIString("Privacy"),
+            settings: [
+                {name: WI.UIString("Enable Intelligent Tracking Prevention debug mode"), setting: InspectorBackend.Enum.Page.Setting.ITPDebugModeEnabled, value: true},
+                // COMPATIBILITY (iOS 14.0): `Page.Setting.AdClickAttributionDebugModeEnabled` was renamed to `Page.Setting.PrivateClickMeasurementDebugModeEnabled`.
+                {name: WI.UIString("Enable Private Click Measurement debug mode"), setting: InspectorBackend.Enum.Page.Setting.PrivateClickMeasurementDebugModeEnabled, value: true},
+
+                // FIXME: Find out whether we need to keep this.
+                // {name: WI.UIString("Ad Click Attribution Debug Mode"), setting: InspectorBackend.Enum.Page.Setting.AdClickAttributionDebugModeEnabled, value: true},
+            ],
+        };
+
+        let mediaSettingsGroup = {
+            name: WI.UIString("Media"),
+            settings: [
+                {name: WI.UIString("Allow media capture on insecure sites"), setting: InspectorBackend.Enum.Page.Setting.MediaCaptureRequiresSecureConnection, value: false},
+                {name: WI.UIString("Disable ICE candidate restrictions"), setting: InspectorBackend.Enum.Page.Setting.ICECandidateFilteringEnabled, value: false},
+                {name: WI.UIString("Use mock capture devices"), setting: InspectorBackend.Enum.Page.Setting.MockCaptureDevicesEnabled, value: true},
+
+                // FIXME: Find out whether we need to keep this.
+                // {name: WI.UIString("Disable Encryption"), setting: InspectorBackend.Enum.Page.Setting.WebRTCEncryptionEnabled, value: false},
```

Comment:
ditto

Reply:
This setting is indeed slated to be removed. It's removal had landed in https://commits.webkit.org/258804@main, but it was then reverted in https://commits.webkit.org/258884@main due to a number of new test failures running API tests against the simulator. That mystery has not yet been reinvestigated yet, and may have been complete happenstance.

That said, we should keep the setting here for now until we can tear out the backend hooks for it, otherwise we are much more likely to punt on removing the backend hooks, leaving behind dead code.

Reply:
Got it. Re-added the "Disable encryption" setting.